dot/sync/tip_syncer.go

```diff
@@ -8,23 +8,29 @@ import (
 	"math/big"
 
 	"github.com/ChainSafe/gossamer/dot/network"
+	"github.com/ChainSafe/gossamer/dot/types"
 	"github.com/ChainSafe/gossamer/lib/common"
 )
 
 var _ workHandler = &tipSyncer{}
 
+type handleReadyBlockFunc = func(*types.BlockData)
```

Comment:
nit does the following work?

```suggestion
type handleReadyBlockFunc func(*types.BlockData)
```

Reply:
yes it does, is there a preference for one?